client/src/components/PatientSearchDateTable/index.js

```diff
@@ -4,6 +4,7 @@ import moment from 'moment';
 import { useLocation } from 'react-router-dom';
 import { bool } from 'prop-types';
 import { message } from 'antd';
+import socketIOClient from 'socket.io-client';
```

Comment:
repeated code 
isn't it better to create a folder IO  and export the functions from it